### DISCUSSION TITLE
Is it possible to connect to cameras that do not directly support NVRs?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#9756] [General]
I have a [VTECH RM7766 baby monitor](https://phones.vtechcanada.com/en/pd/4710) and was wondering if there was a way to "hack" the stream to connect to frigate? I don't think VTECH provides direct support for NVRs, and I don't think it supports RTSP (not sure tho). There are no web interfaces at the IPs of the camera nor the "tablet" that comes with it. The system is able to connect to the router and stream remotely, so some sort of IP is supported. What technologies do these "closed" systems typically use behind the scenes? Is it possible to reverse engineer it and plug it into Frigate?

**Comment by mattr9** [comment]
sorry I can’t help but I’m interested in the same answer. Did you figure it out?

**Reply by thinkloop** [reply]
I did not figure it out, sorry 😔
